funnel/templates/schedule_edit.html.jinja2

```diff
@@ -133,7 +136,7 @@
       var ROOMS = {{ rooms|tojson }};
       var BREAK_EVENTS_COLOR = "#999999";
       var EDIT_EVENTS = true;
-      var CALENDAR_SLOT_MINUTES = 5;
+      var CALENDAR_SLOT_MINUTES = 30;
```

Comment:
Is there a way to manually edit times to choose a smaller interval? Zainab loves her 5 minute sessions.

Reply:
Have added buttons to switch between 5m, 15m, 30m, 60m intervals